shared/src/main/kotlin/com/egm/stellio/shared/model/ApiExceptions.kt

```diff
@@ -2,9 +2,11 @@ package com.egm.stellio.shared.model
 
 import com.apicatalog.jsonld.JsonLdError
 import com.apicatalog.jsonld.JsonLdErrorCode
+import com.egm.stellio.shared.util.NGSILDWarning
 
 sealed class APIException(
-    override val message: String
+    override val message: String,
+    var warnings: List<NGSILDWarning>? = null
```

Comment:
let's discuss about this, don't really like seeing the warnings "polluting" the whole error hierarchy (whereas it is specific to distOps and HTTP binding)